Src/VsVimShared/Implementation/Misc/FallbackKeyProcessor.cs

```diff
@@ -200,19 +221,53 @@ internal bool TryProcess(KeyInput keyInput)
             // take place in this scenario
             if (_vsShell.IsInModalState())
             {
+                _firstChord = null;
                 return false;
             }
 
             // Check for any applicable fallback bindings, in order
             VimTrace.TraceInfo("FallbackKeyProcessor::TryProcess {0}", keyInput);
-            foreach (var fallbackCommand in _fallbackCommandList)
+            var findFirstChar = _firstChord != null ? _firstChord.Char : keyInput.Char;
+            var findFirstModifiers = _firstChord != null ? _firstChord.KeyModifiers : keyInput.KeyModifiers;
+            var cmds = _fallbackCommandList[findFirstChar]

```

Comment:
The _fallbackCommandList is an ILookup so if nothing is found an empty IEnumerable<FallbackCommand> is returned.